hyperlane/types/src/va.rs

```diff
@@ -37,6 +39,9 @@ pub enum QueryMsg {
     AnnouncedStorageLocations {
         validators: BTreeSet<HexByteArray<20>>,
     },
+    /// Query the cost of announcing a storage location.
+    #[returns(Coin)]
+    CalculateAnnounceFee { storage_location: String },
```

Comment:
query function name should be a noun phrase, not a verb, to be consistent with all the other methods above. so just `AnnounceFee`, not `CalculateAnnounceFee`